src/main/java/frc/robot/constants/RobotConfig.java

```diff
@@ -85,21 +91,28 @@ public static final class ShooterConfig {
     public static final double kDefaultFlywheelRPM = 1000;
 
     public static final double kShooterStowAngle = 0;
+    public static final Measure<Angle> kZeroedAngle = Units.Rotations.of(28.6 / 360 * 160);
```

Comment:
What does this math here represent? Should this be done using a unit library?

Reply:
percentage of max rotations, fine for now